app/facades/food_facade.rb

```diff
@@ -0,0 +1,11 @@
+class FoodFacade
+  def self.search(search_params)
+    json = FoodService.search_foods(search_params)
+    foods =json[:foods].map { |food_info| Food.new(food_info) }
+    foods[0..9]
+  end
+  def self.total_results(search_params)
+    json = FoodService.search_foods(search_params)
+    json[:totalHits]
```

Comment:
There is a great opportunity for memoization here. Something like: 
```ruby
def self.search_foods(search_params)
json ||= FoodService.search_foods(search_params)
end
```

This way you would only ever hit the API once when you call this method and instead use the memoized response when you need to call that method again.